.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
this is a new project and 16 is getting out of support in September. Can we bump this (i.e. to 20)?

Reply:
Looks like we can try: https://community.grafana.com/t/which-node-version-to-use/34347/3

But @ollidraese I tried it, and found that it's going to be a slippery slope with needing to upgrade dependencies.  This was the first one on the slope:

```
11:06:38 in cribl-search-grafana-plugin on initial-version [!] on 🐳 v24.0.2 ✘1 ➜ yarn install
yarn install v1.22.19
warning ../../package.json: No license field
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
error eslint-plugin-jsdoc@39.6.2: The engine "node" is incompatible with this module. Expected version "^14 || ^16 || ^17 || ^18 || ^19". Got "20.5.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

I think we should get this plugin out the door with 16, and maybe followup with a 20 upgrade shortly down the road.

Reply:
Upgraded to Node v18